modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -514,7 +513,7 @@ public static void createBinding(Exchange exchange, AMQShortString routingKey,
             Andes.getInstance().addBinding(binding);
         } catch (AndesException e) {
             log.error("error while creating binding", e);
```

Comment:
Logger will print the exception message when exception object is passed in. If we add e.getMessage() to the message it will be redundant and hence is not a good practice.